bots/elasticsearch/tasks.py

```diff
@@ -18,6 +21,22 @@ def safe_substr(value, length=32000):
         return str(value)[:length]
     return None
 
+def add_suggest(obj):
+    input = obj['name']
+    if input:
+        # Elastic's completion suggester only looks at prefixes. If we give each
```

Comment:
Yeah, exactly (except with the full string `'Foo Bar Baz'` in there, too).

I think if we just split on whitespace, it would fail to match if someone types more than one word (e.g. `university of ...`).